go.mod

```diff
@@ -9,24 +9,54 @@ require (
 	github.com/eclipse/paho.mqtt.golang v1.3.5
 	github.com/gorilla/mux v1.8.0
 	github.com/oklog/ulid/v2 v2.0.2
-	github.com/project-alvarium/alvarium-sdk-go v0.0.0-20240328184249-0381aa4861f0
+	github.com/project-alvarium/alvarium-sdk-go v0.0.0-20240705181105-e5ec0811a099
 	go.mongodb.org/mongo-driver v1.8.4
 )
 
+require github.com/modern-go/reflect2 v1.0.2 // indirect
```

Comment:
Is this package needed anywhere ? I tried building without it and the build went through, let me know if I am wrong.

Reply:
It keeps coming whenever I run `go mod tidy`, I haven't used it directly in my code.
![image](https://github.com/user-attachments/assets/2e3ae214-9c4a-418f-a4e6-4d65fc1ff2b5)